packages/web-components/src/components/va-breadcrumbs/va-breadcrumbs.tsx

```diff
@@ -25,7 +28,35 @@ export class VaBreadcrumbs {
    * Adds an aria-label attribute to the <nav /> element.
    */
   @Prop() label?: string = 'Breadcrumb';
-
+  /**
+     * Whether or not the component will use USWDS v3 styling.
+     */
+  @Prop() uswds?: boolean = false;
+  /**
+   *  Whether or not the component will wrap the breadcrumbs. This prop is available when `uswds` is set to `true`.
+   */
+  @Prop() wrapping?: boolean = false;
+  /**
+   *  Represents a list of breadcrumbs. Use an array of objects with label and href properties, and then use JSON.stringify() to convert to a string. This prop is available when uswds is set to true.
+   */
+  @Prop() breadcrumbList?: any;
+  /**
+   * 
+   * Represents an internal state of the component which stores the list of breadcrumbs parsed from the 'breadcrumbList' prop. 
+   * Each breadcrumb is represented as an object with two properties: 'label' and 'href'.
+   */
+  @State() formattedBreadcrumbs?: Array<{ label: string; href: string }> = [];
+  /**
+   * 
+   * This is a method that watches for changes in the 'breadcrumbList' prop. 
+   * When the 'breadcrumbList' prop changes, this method parses the new value (provided as a JSON-formatted string) 
+   * into a JavaScript object and assigns it to the 'breadcrumbsState' state.
+   */
+  @Watch('breadcrumbList')
```

Comment:
Are `@State()` and `@Watch` only used for `uswds`? If so, can you just make a mention in the comments that this is for uswds? That might help us with maintaining the code in the future.